File: JavaBasic/122_all_primes_100.java
// 122	122_all_primes_100.java	Print all prime numbers from 1 to 100.
public class _122_all_primes_100 {
    public static void main(String[] args) {
        System.out.print("Primes 1-100: ");
        for (int n = 2; n <= 100; n++) {
            boolean p = true;
            for (int i = 2; i <= Math.sqrt(n); i++) if(n%i==0){p=false;break;}
            if (p) System.out.print(n + " ");
        }
        System.out.println();
    }
}

File: JSBASIC/122_all_primes_100.js
// 122	122_all_primes_100.js	Print all prime numbers from 1 to 100.

console.log("Prime numbers from 1 to 100:");
for (let num = 2; num <= 100; num++) {
    let isPrime = true;
    for (let i = 2; i <= Math.sqrt(num); i++) {
        if (num % i === 0) { isPrime = false; break; }
    }
    if (isPrime) process.stdout.write(num + " ");
}
console.log();
